website/templates/log_templates.mako

```diff
@@ -37,9 +37,8 @@ initiated an embargoed registration of
 </script>
 
 <script type="text/html" id="embargo_terminated_no_user">
-Embargo for
-<a class="log-node-title-link overflow" data-bind="text: nodeTitle, attr: {href: nodeUrl}"></a>
-ended.
+Embargo of registration of
```

Comment:
I think we're reverting this language change, so yeah, just the `.` and fixing the typos in the docstring above should do it.